db/migrate/20151028061513_metacode_asset_path_update.rb

```diff
@@ -0,0 +1,10 @@
+class MetacodeAssetPathUpdate < ActiveRecord::Migration
+  def change
+    Metacode.all.each do |metacode|
+      if metacode.icon.start_with?("/assets/icons/")
+        metacode.icon = metacode.icon.gsub(/^\/assets\/icons/, "https://s3.amazonaws.com/metamaps-assets/metacodes")
```

Comment:
This won't work for us yet. I think the thing to do might actually be do it manually through the interface, because there isn't any particularly good automation for it, that wouldn't take as long as doing it this way.